packages/base/source/server/errors.js

```diff
@@ -12,4 +11,4 @@ Space.Error.extend(Todos, 'TodoNotFoundError', {
     Space.Error.constructor.call(this);
     this.message = `No todo with id ${todoId} found.`;
```

Comment:
Hey guys, it's always like the snippet i posted you last time:
```javascript
Space.Error.extend(Todos, 'TodoNotFoundError', {
  Constructor(todoId) {
    Space.Error.constructor.call(this);
    this.message = `No todo with id ${todoId} found.`;
  }
});
```
You have to call the super constructor with the message, otherwise this is not a proper error class!